apps/fmr/index.js

```diff
@@ -0,0 +1,56 @@
+const hof = require('hof');
+const Summary = hof.components.summary;
+const clearSession = require('./behaviours/clear-session');
```

Comment:
You already import the HOF clearSession component in hof.settings.json which can be used with `clearSession: true` on the last step: https://github.com/UKHomeOffice/additional-security-checks/blob/master/apps/asc/index.js#L691

If you prefer this behaviour then no problem